src/addRemove.js

```diff
@@ -1,34 +1,44 @@
 // eslint-disable-next-line import/no-cycle
```

Comment:
- [ ]  [optional] please remove all the unused commented code in order to keep the file clean and pro